scripts/WORKFLOW-EXAMPLES.md

```diff
@@ -0,0 +1,205 @@
+# Job examples
```

Comment:
Is the `ALL_UPPERCASE.md` filename a convention for MD files somehow?  Don't care much either way but wondered about it...

Reply:
No - usually the READE.md files are.